### ISSUE TITLE
Launch doesn' when in ssh

### ISSUE THREAD

**Issue by mpauper** [kovidgoyal/kitty#7062]
**Describe the bug**
Hello. I have the following keybinds set:

**To Reproduce**
1. XXX
2. YYY
3. ZZZ

**Screenshots**

**Environment details**
```
Press Ctrl+Shift+F6 (cmd+option+comma on macOS) in kitty, to copy debug output about kitty and its
configuration to the clipboard and paste it here. 

On older versions of kitty, run kitty --debug-config instead
```

**Additional context**
Try to reproduce the problem with `kitty --config NONE` if you cannot then post a minimal kitty.conf that reproduces the problem. If the problem involves interaction with some other terminal program post a minimal config for that program to reproduce the problem as well.

**Comment by kovidgoyal** [comment]
This is an empty bug report, on the off chance that it was supposed to be about some keybinding not working, take a look at the existing fixed regressions in 0.32 with regards to keybindings.

**Comment by mpauper** [comment]
> This is an empty bug report

Yes, sorry for that, submitted before finishing the write up. I then messed up the formatting. Apologies for the mess.

> on the off chance that it was supposed to be about some keybinding not working, take a look at the existing fixed regressions in 0.32 with regards to keybindings.

I am using 0.32 and in fact I feel like these keybinds used to work by following ssh connection until I updated to 0.32.

**Comment by kovidgoyal** [comment]
Wait for 0.32.1 or revert to 0.31 in the meantime.
